Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -135,16 +135,16 @@ def bump_formula_pr
     formula = args.named.to_formulae.first
 
     new_url = args.url
-    formula ||= determine_formula_from_url(new_url) if new_url
-    raise FormulaUnspecifiedError unless formula
+    formula ||= determine_formula_from_url(new_url) if new_url.present?
+    raise FormulaUnspecifiedError if formula.nil?
```

Comment:
```suggestion
    raise FormulaUnspecifiedError if formula.present?
```

to be more properly equivalent

Reply:
`unless formula` != `if formula.present?`

Reply:
Yes, sorry.